packages/launcher/src/main/index.js

```diff
@@ -259,6 +264,13 @@ const setupClient = async () => {
   }
 
   // Start daemon and connect local client to it
+  try {
+    await shutdownDaemon()
```

Comment:
Any reason for this please?
cc @howleysv this seems unnecessary following your changes, no?

Reply:
@howleysv 
The reason for this is that if you close the launcher with Ctrl+C the
daemon is still running. And it fails to connect.

I can remove it ... but it’s a big we need to fix anyways

Reply:
Yes the reason I didn't do this for dev env was because it breaks the flow of running the daemon separately from the launcher (which I believe is the recommended method for running MainframeOS in dev env).

TBH I'd rather just not ever try start the daemon inside electron in dev envs.

Reply:
OK makes sense, so better add this logic of starting/stopping the daemon only in prod build, and in dev document running the daemon separately.

Reply:
It is already the case in prod. The change just duplicates the logic of the lines above it without the env restriction

Reply:
@howleysv Are you suggesting this:

```
 if (envType === 'production') {
    daemonConfig.runStatus = 'stopped'
    await stopDaemon(env.name)

    // Start daemon and connect local client to it
    await startDaemon(env.name)
    localDaemon = true
  }

```

Reply:
Anyways.. I'm removing that from my PR
I put just because it was big pain while I was developing..

Reply:
That is the logic already

Reply:
Oh you mean what I think it should be? Not exactly that, but in terms of handling the daemon running inside electron, the idea would be similar